test/integration/Highlight.e2e.test.js

```diff
@@ -0,0 +1,30 @@
+// <reference types="Cypress" />
+describe('Highlight', () => {
```

Comment:
Should this be plural to match `Regions`?

Reply:
Next time I promise